deployment/common/common.yaml

```diff
@@ -21,4 +21,77 @@ spec:
   scope: Namespaced
   names:
     plural: routes
-    kind: Route
\ No newline at end of file
+    kind: Route
+  validation:
+    # openAPIV3Schema is the schema for validating custom objects.
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+            - host
+          properties:
+            strategy:
+              type: string
+              enum:
+                - RoundRobin
+                - WeightedLeastRequest
+                - Random
+            lbHealthCheck:
+              type: object
+              properties:
+                path:
+                  type: string
+                  pattern: ^/(\w+)?$
```

Comment:
Does this pattern allow `/maps/foo`?

Reply:
hmm probably not, good point, let me refactor